Makefile

```diff
@@ -1,10 +1,13 @@
-.PHONY: build generate test clean
-build:generate
-	go build -o output/pb-go main/main.go
+.PHONY: build generate test clean server client
+build:generate server client
 generate:
 	go generate main/main.go
 test:
 	go test -coverprofile=output/coverage.out ./...
 clean:
 	rm -rf output
 	rm -rf templates/*.qtpl.go
+server:
+	go build -o output/pb-go main/main.go
+client:
+	# todo: build pg-go-cli
```

Comment:
Client go source file path is already defined in the upstream, the server source file path also changed. Please pull upstream and make some changes here. Thanks.

Reply:
Let's just call client `pb-cli`! OK?

Reply:
Perfect. Also, you need to add another action for `release` process when a new git tag is pushed......

Reply:
OK

Reply:
I will create another workflow later.